Presentation.Web/Controllers/API/V2/External/ItContracts/ItContractV2Controller.cs

```diff
@@ -373,9 +374,29 @@ public IHttpActionResult DeleteExternalReference([NonEmptyGuid] Guid contractUui
                 .Match(_ => NoContent(), FromOperationError);
         }
 
+        [HttpGet]
+        [Route("applied-procurement-plans/{organizationUuid}")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(IEnumerable<AppliedProcurementPlanResponseDTO>))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        public IHttpActionResult GetAppliedProcurementPlans([NonEmptyGuid] [FromUri] Guid organizationUuid)
```

Comment:
move it to the internal controller. 

We don't want to expose this endpoint to API users